docs/opa-plugins-architecture.md

```diff
@@ -8,36 +8,57 @@ This document describes the high-level flow of interactions and the sequence dia
 graph LR
     A[Backstage] --> B[Permissions Framework]
     B --> C[OPA Permissions Wrapper Plugin]
-    C --> D[OPA Backend Plugin]
-    D --> E[OPA]
-    E --> D
+    C --> D[OPA]
     D --> C
     C --> B
     B --> A
 ```
 
-## Sequence Diagram of the Plugin Interactions
+## Sequence Diagram of the Plugin Interactions (Old Backend)
 
 This flowchart represents the high-level flow of interactions between the different components in the OPA plugins architecture.
 
 ```mermaid
-
 sequenceDiagram
     participant User
     participant Backstage App
     participant permissions.ts
-    participant policyEvaluator
-    participant OpaClient
+    box "permission-backend-module-opa-wrapper"
+        participant policyEvaluator
+        participant OpaClient
+    end
     participant OPA Server
 
     User->>Backstage App: Makes a request
     Backstage App->>permissions.ts: Forward request
     permissions.ts->>policyEvaluator: evaluatePolicy(permissionRequest)
     policyEvaluator->>OpaClient: evaluatePolicy(permissionRequest)
-    OpaClient->>OPA Server: POST /api/opa/opa-permissions/{opaPackage}
+    OpaClient->>OPA Server: POST /v1/data/{opaPackage}
     OPA Server-->>OpaClient: PolicyEvaluationResult
     OpaClient-->>policyEvaluator: Return PolicyEvaluationResult
     policyEvaluator-->>permissions.ts: Return evaluation result
     permissions.ts-->>Backstage App: Return evaluation result
     Backstage App-->>User: Return response
 ```
+
+## Sequence Diagram of the Plugin Interactions (New Backend)
```

Comment:
This one needs a bit of tweaking still, need to display the permission framework and its involvement here :)